src/data/sofia-districts-names.js

```diff
@@ -26,4 +26,29 @@ export const districtNames = new Map([
 ])
 
 
-//
\ No newline at end of file
+export const districtOptions = [
+{ value:"Средец", label: "Средец" },
```

Comment:
' symbol was used for the topics and " used here. Spacing between "value:" and the district.